### DISCUSSION TITLE
Feature Request: "Crop by colour"

### DISCUSSION THREAD

**Discussion by Kazuto** [spatie/image#132] [Ideas]
Hello guys,

I'd like to request a feature :)

Since we're able to add a border I think it would be useful to be able to remove a border or rather crop by a given colour.
Imagine having an image where there are letterboxes or pillar boxes. It would be great to add the given colour to the crop manipulation and be removed. Basically like GD's `imagecropauto`.

What do you guys think?

**Comment by petsoukos** [comment]
+1
